tests/specs/functional/infinite-gas-spec.k

```diff
@@ -98,8 +113,8 @@ module INFINITE-GAS-SPEC
 
     claim <k> runLemma(Gverylow < SCHED > <=Int #gas(_)) => doneLemma(true) ... </k>
 
-    claim <k> runLemma(Gexp < SCHED > +Int Gexpbyte < SCHED > *Int ( log2Int ( W1 ) /Int 8 +Int 1 ) <=Int #gas(GAVAIL)) => doneLemma(true)  ... </k>
-    claim <k> runLemma(#gas(GAVAIL) <Int Gexp < SCHED > +Int Gexpbyte < SCHED > *Int ( log2Int ( W1 ) /Int 8 +Int 1 ))  => doneLemma(false) ... </k>
+    claim <k> runLemma(Gexp < SCHED > +Int Gexpbyte < SCHED > *Int ( log2Int ( W1 ) /Int 8 +Int 1 ) <=Int #gas(GAVAIL)) => doneLemma(true)  ... </k> requires 0 <Int W1 andBool W1 <Int pow256
+    claim <k> runLemma(#gas(GAVAIL) <Int Gexp < SCHED > +Int Gexpbyte < SCHED > *Int ( log2Int ( W1 ) /Int 8 +Int 1 ))  => doneLemma(false) ... </k> requires 0 <Int W1 andBool W1 <Int pow256
```

Comment:
Why not #rangeUInt on these requires?

Reply:
Because the bottom of the range is `0 <Int W1` instead of `0 <=Int W1`. But this reminds me that these tests can be simplified slightly now.

Reply:
But it's not necessary to simplify them I guess.

Reply:
Sounds good. I approved.